### PR TITLE
fix(neon_framework): Fix Android push notification icon

### PR DESCRIPTION
**Pull request by provokateurin**
Before/after:
image

### CODE REVIEW

File: packages/neon_framework/test/push_utils_test.dart [hunk 1]
```diff
@@ -119,7 +119,11 @@ void main() {
       FlutterLocalNotificationsPlatform.instance = localNotificationsPlatform;
       when(
         () => localNotificationsPlatform.initialize(
-          any(),
+          any(
+            that: predicate(
```
Comment (Leptopoda) [inline packages/neon_framework/test/push_utils_test.dart:123]: Please use a type matcher instead of a predicate

Comment (provokateurin) [reply packages/neon_framework/test/push_utils_test.dart:123]: That's not possible: https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/lib/src/platform_specifics/android/initialization_settings.dart

Comment (Leptopoda) [reply packages/neon_framework/test/push_utils_test.dart:123]: I don't get why.
```dart
any(
isA<Type>().having((e) => e.attribute, "attribute", "expected")
)
```

Should work the same and produce better failure messages.
I just noticed that I missed this entirely in my last review and all other tests also make use of `predicate` matchers so let's just continue with it here.

**Review by Leptopoda — APPROVED** [review]

File: packages/neon_framework/test/push_utils_test.dart (hunk 1 above)
Comment (provokateurin) [reply packages/neon_framework/test/push_utils_test.dart:123]: Ah I thought you wanted to have it compared to another instance of AndroidInitializationSettings directly.

**Pull request merged**